src/components/Text/Text.module.css

```diff
@@ -55,6 +55,10 @@ button.text.link:focus-visible {
 }
 
 /* font-size */
+.body.xs {
+  --font-size: var(--text-body-xs-size);
```

Comment:
The leading of the body does not vary from one size to another, so it is specified as a batch.